ops/mappings.json

```diff
@@ -534,20 +534,6 @@
         "mapping": "ContributionReward",
         "arcVersion": "0.0.1-rc.34"
       },
-      {
-        "name": "UGenericScheme",
-        "contractName": "UGenericScheme",
-        "dao": "base",
-        "mapping": "UGenericScheme",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "SchemeRegistrar",
```

Comment:
why this is removed ?

Reply:
It isn't. I just moved it down a bit, git showing it weirdly